### PR TITLE
fix: change k8s port to 5800

### PR DESCRIPTION
**Pull request by tcp**
## Description
Avoid Mac conflicts

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by yashgorana** [comment]
Closes https://github.com/OpenMined/Heartbeat/issues/979

File: tox.ini [hunk 1]
```diff
@@ -735,6 +736,9 @@ commands =
 
     sleep 30
 
+    # Patches CoreDNS
+    tox -e dev.k8s.patch.coredns
```
Comment (yashgorana) [inline tox.ini]: We can call patch before the `sleep 30` and right after cluster testdomain. This way the config is available during before the deploy where it will be used. Else coredns will only refresh every 15s

File: docs/source/deployment/index.rst [hunk 2]
```diff
@@ -380,7 +380,7 @@ Ingress is not working on Mac and Docker and the issue is `being tracked here <h
 
     $ docker ps
     CONTAINER ID   IMAGE                                 COMMAND                  CREATED        STATUS              PORTS                                                                                                                                  NAMES
-    57f73851bf08   gcr.io/k8s-minikube/kicbase:v0.0.25   "/usr/local/bin/entr…"   46 hours ago   Up About a minute   127.0.0.1:57954->22/tcp, 127.0.0.1:57955->2376/tcp, 127.0.0.1:57957->5000/tcp, 127.0.0.1:57958->8443/tcp, 127.0.0.1:57956->32443/tcp minikube
+    57f73851bf08   gcr.io/k8s-minikube/kicbase:v0.0.25   "/usr/local/bin/entr…"   46 hours ago   Up About a minute   127.0.0.1:57954->22/tcp, 127.0.0.1:57955->2376/tcp, 127.0.0.1:57957->5800/tcp, 127.0.0.1:57958->8443/tcp, 127.0.0.1:57956->32443/tcp minikube
```
Comment (yashgorana) [inline docs/source/deployment/index.rst]: unrelated change, we can revert this

**Review by yashgorana — COMMENTED** [review]
Awesome work @tcp 💪🚀 Just minor comments and we can merge this once 0.8.4 is released.

File: docs/source/deployment/index.rst (hunk 2 above)
Comment (tcp) [reply docs/source/deployment/index.rst]: `sed` things :)

**Review by yashgorana — APPROVED** [review]
Ah I see that 0.8.4 branch is already created, so we can go ahead and merge this.

**Comment by rasswanth-s** [comment]
Yay! Finally Mac Support , Great work @tcp
